backend/libraries/timer_job_queues/src/grouped_timer_job_queue.rs

```diff
@@ -197,6 +197,12 @@ where
     }
 }
 
+impl<T: TimerJobItemGroup> Default for GroupedTimerJobQueue<T> {
+    fn default() -> Self {
+        GroupedTimerJobQueue::new(5, true)
```

Comment:
That's quite an opinionated default. Previously the dev would have to think out the max concurrency which I think is better than having it set for them.